src/System.Net.Http/src/System/Net/Http/Headers/MediaTypeHeaderValue.cs

```diff
@@ -262,7 +262,8 @@ private static void CheckMediaTypeFormat(string mediaType, string parameterName)
             int mediaTypeLength = GetMediaTypeExpressionLength(mediaType, 0, out tempMediaType);
             if ((mediaTypeLength == 0) || (tempMediaType.Length != mediaType.Length))
             {
-                throw new FormatException(string.Format(System.Globalization.CultureInfo.InvariantCulture, SR.net_http_headers_invalid_value, mediaType));
+                // This fails when running in an net45 environment, but compiled in an net46 environment, when last parameter is not an object array
+                throw new FormatException(string.Format(System.Globalization.CultureInfo.InvariantCulture, SR.net_http_headers_invalid_value, new object[] { mediaType }));
```

Comment:
@davidsh why using Invariant culture to format this exception message? if it is not necessary, I would recommend just using String.Format(string, object) which should work on net45 too.